src/game/behaviors/clock_arm.inc.c

```diff
@@ -3,20 +3,93 @@
  */
 
 #include "sm64ap.h"
+#include "../../sm64ap.h"
 
 void bhv_rotating_clock_arm_loop(void) {
-    if (o->oBehParams != SM64AP_CourseToTTC()) {
+    int entrance = SM64AP_EntranceToTTC();
+    int course = entrance / 10;
+    s32 isMinuteHand = cur_obj_has_behavior(bhvClockMinuteHand);
```

Comment:
Why did you introduce this variable? As shorthand? If so, please revert

Reply:
Fixed in 0bd6347f64303e68b1355d9a55b7ac620f12ffd7. Additional commits have been added to ensure code changes are absolute minimum needed to ensure functionality